environment.yml

```diff
@@ -0,0 +1,30 @@
+name: aas229-workshop
+
+channels:
+  - http://ssb.stsci.edu/astroconda
+
+dependencies:
```

Comment:
Comments on version numbering are the same as above in `check_env.py`. I think all the files should be consistent.

Also, what about `specviz` and `stginga`?